benchmarking/smartseq2/compare_matrix/run_matrix_comparison.wdl

```diff
@@ -1,18 +1,20 @@
 task CompareDataMatrix {
   File base_datafile
   File updated_datafile
-  File gtf_file
+  File metadata_file
   String output_name
   String src_dir
   
   command {
     set -e
     mkdir ${output_name}
     gsutil cp ${src_dir}/Compare_data_matrix.R ./
+    gsutil cp ${src_dir}/analysis_functions.R ./
```

Comment:
Eventually it would be good to have these scripts (especially this shared functions file) in a docker.